platform/ui/src/contextProviders/DialogProvider.js

```diff
@@ -114,62 +125,84 @@ const DialogProvider = ({ children, service }) => {
     });
   };
 
+  const validCallback = callback => callback && typeof callback === 'function';
+
   return (
-    <DialogContext.Provider value={{ create, dismiss, dismissAll, dialogs }}>
+    <DialogContext.Provider value={{ create, dismiss, dismissAll, noDialogs }}>
       <div className="DraggableArea">
         {dialogs.map(dialog => {
           const {
-            id,
-            content: Dialog,
-            position /* Position of the dialog. {{x: 0, y: 0}} */,
-            defaultPosition,
+            id = null,
+            content: DialogContent = null,
+            contentProps = null,
+            position = null,
+            defaultPosition = null,
             isDraggable = true,
-            onStart = () => {},
-            onStop = () => {},
-            onDrag = () => {},
+            showOverlay = false,
```

Comment:
If each dialog can `showOverlay`, what happens if multiple do?

Reply:
Nothing, the user will have to answer one at a time according to the call order.